duct_test.go

```diff
@@ -367,8 +367,55 @@ func TestLoggerOption(t *testing.T) {
 	}
 
 	logs := buffer.String()
-	t.Log(logs)
 	if len(logs) == 0 {
 		t.Fatal("Didn't capture logs")
 	}
+
+}
+
+func TestWaitForExit(t *testing.T) {
+
+	c := New(Manifest{
+		{
+			Name:        "Exit",
+			Command:     []string{"sleep", "1"},
+			Image:       "debian:latest",
+			WaitForExit: true,
+		},
+	}, WithNewNetwork("duct-test-network"))
+
+	if err := c.Launch(context.Background()); err != nil {
+		t.Fatal(err)
+	}
+
+	if err := c.Teardown(context.Background()); err != nil {
+		t.Fatal(err)
+	}
+
+	c = New(Manifest{
+		{
+			Name:        "ExitBadly",
+			Command:     []string{"ls", "-123"},
```

Comment:
let's make this:

```go
[]string{"sh", "-c", "exit 1"}
```

Depending on broken arguments to `ls` seems a bit brittle. Likewise, let's make the `sleep 1` do `exit 0` instead, it'll speed things up.